src/box/flightrec.h

```diff
@@ -67,6 +67,17 @@ box_set_flightrec(void)
 	return 0;
 }
 
+/**
+ * This function is called in SIGBUS handler to check whether accessed address
+ * belongs to flightrec file.
+ */
+static inline bool
```

Comment:
Nit: Let's add `stdbool.h` to the header list.

Reply:
Done.